### PR TITLE
Form submission validation

### PR DESCRIPTION
**Pull request by ahmedhamodi**
Error message examples are below.

Violating word count (max and min):
image

Violating char count (max and min):
image

### CODE REVIEW

**Comment by ahmedhamodi** [comment]
Last bug: on initial form submission load, errors are not detected and user is able to move to next page.

File: src/Components/FormCreation/Submission/CreateSubmissionForm.js [hunk 1]
```diff
@@ -302,6 +303,20 @@ function CreateSubmissionForm({ match }) {
     });
   }, [page]);
 
+  const handleNext = () => {
+    // TODO: check validation of all elements on page
+    // TODO: check that all required questions are answered
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/Submission/CreateSubmissionForm.js:308]: Remove

Comment (ahmedhamodi) [reply src/Components/FormCreation/Submission/CreateSubmissionForm.js:308]: Make the whole function back as an inline ()=>setPage(page+1)

File: src/Components/FormCreation/Submission/CreateSubmissionForm.js [hunk 2]
```diff
@@ -438,9 +455,9 @@ function CreateSubmissionForm({ match }) {
                 <Button
                   variant="outlined"
                   className={classes.button}
-                  disabled={page === sections.length - 1}
+                  disabled={page === sections.length - 1 || isValid !== 0}
                   color="primary"
-                  onClick={() => setPage(page + 1)}
+                  onClick={() => handleNext()}
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/Submission/CreateSubmissionForm.js:460]: Revert this

**Review by GodGreg — APPROVED** [review]

**Pull request merged**